compiler/semstmts.nim

```diff
@@ -2118,7 +2118,7 @@ proc semStmtList(c: PContext, n: PNode, flags: TExprFlags): PNode =
             continue
 
         let verdict = semConstExpr(c, n[i])
-        if verdict.intVal == 0:
+        if verdict == nil or verdict.kind != nkIntLit or verdict.intVal == 0:
```

Comment:
I have found that megatest doesn't compile with debug nim (verdict is not integer node is some of the tests hence I had to fix it)